### DISCUSSION TITLE
nice egui tutorial video

### DISCUSSION THREAD

**Discussion by BctfN0HUK7Yg** [emilk/egui#936] [Show and tell]
Getting started with Rust 🦀 2021: 7a. Building a GUI app in Rust [Part A]
https://www.youtube.com/watch?v=NtUkr_z7l84

Getting started with Rust 🦀 2021: 7b. Building a GUI app in Rust [Part B]
https://www.youtube.com/watch?v=SvFPdgGwzTQ

**Comment by emilk** [comment]
This is great, thanks for linking to these!
